hvac/v1/__init__.py

```diff
@@ -744,8 +744,9 @@ def list_vault_ec2_certificate_configurations(self):
         return self._get('/v1/auth/aws-ec2/config/certificates', params=params).json()
 
     def create_ec2_role(self, role, bound_ami_id=None, bound_account_id=None, bound_iam_role_arn=None,
-                        bound_iam_instance_profile_arn=None, role_tag=None, max_ttl=None, policies=None,
-                        allow_instance_migration=False, disallow_reauthentication=False, **kwargs):
+                        bound_iam_instance_profile_arn=None, role_tag=None, ttl=None, max_ttl=None,
```

Comment:
I don't have a obvious suggestion for how to avoid this, but its worth nothing that adding in these new kwargs in the middle of the method's parameters will break backwards compatibility in some cases. I'm suppose that is something we can just note in the changelog 🤷‍♂️.